src/plots/linePlot.ts

```diff
@@ -21,10 +23,121 @@ export module Plots {
       this.attr("stroke-width", "2px");
     }
 
+    public y(): Plots.AccessorScaleBinding<number, number>;
+    public y(y: number | Accessor<number>): Plots.Line<X>;
+    public y(y: number | Accessor<number>, yScale: Scale<number, number>): Plots.Line<X>;
+    public y(y?: number | Accessor<number> | number | Accessor<number>, yScale?: Scale<number, number>): any {
+      if (yScale instanceof QuantitativeScale) {
+        (<QuantitativeScale<number>>yScale).niceDomain(!this._autorangeSmooth);
```

Comment:
One risk with this auto-set-nice feature is that if there are multiple `Plots.Line` depending on the same y-scale, then `autorangeSmooth()` has to be set on all of them individually, otherwise the order of assembly matters.